NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(add_ws_to_parse_data)
```

Comment:
Were these used by the vignette? Does `R CMD check` complain about `:::` in the vignette?